### PR TITLE
doc: Create a README file explaining the setup and sample usage with LangChain and LangGraph.

### PR DESCRIPTION
**Pull request by anubhav756**

### CODE REVIEW

File: sdks/langchain/README.md [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+
+## Development
+
+Setting up a Development Environment:
+
+1. Clone the repository:
+    ```bash
+    git clone https://github.com/googleapis/genai-toolbox.git
+    ```
+1. Navigate to the SDK directory:
```
Comment (twishabansal) [inline sdks/langchain/README.md]: We should be navigating to `genai-toolbox/sdks/langchain` instead

Comment (anubhav756) [reply sdks/langchain/README.md]: Fixed. Thanks for pointing this out!

File: sdks/langchain/README.md [hunk 2]
```diff
@@ -0,0 +1,148 @@
+# GenAI Toolbox SDK
+
+This SDK allows you to seamlessly integrate the functionalities of [Toolbox](https://github.com/googleapis/genai-toolbox) into your LLM applications, enabling advanced orchestration and interaction with GenAI models.
+
+> [!NOTE]
+> Currently the SDK only supports [LangChain](https://python.langchain.com/docs/introduction/) orchestrations.
+> Support for [LlamaIndex](https://docs.llamaindex.ai/en/stable/getting_started/starter_example/) coming soon.
```
Comment (kurtisvg) [inline sdks/langchain/README.md]: nit: don't need to mention this. 
```suggestion
```

File: sdks/langchain/README.md [hunk 3]
```diff
@@ -0,0 +1,148 @@
...
+
+## Installation
+
+You can install the Toolbox SDK for LangChain using `pip`.
+
+```bash
+pip install toolbox-langchain-sdk
```
Comment (kurtisvg) [inline sdks/langchain/README.md:30]: This seems like how I might install it once we push it to pypi. How do I install it today?

File: sdks/langchain/README.md [hunk 4]
```diff
@@ -0,0 +1,148 @@
...
+
+## Load individual tools
+
+Alternatively, you can also load individual tools.
+
+```python
+tool1 = await toolbox.load_tool("tool-1")
+tool2 = await toolbox.load_tool("tool-2")
+# ... and so on
+```
```
Comment (kurtisvg) [inline sdks/langchain/README.md]: @twishabansal mentioned this doesn't work? Should we exclude this section for now?

File: sdks/langchain/README.md [hunk 5]
```diff
@@ -0,0 +1,148 @@
...
+
+## Use with LangChain
+
+LangChain's agents can dynamically choose and execute tools based on the user input. The user can include the tools loaded from the Toolbox SDK in the agent's toolkit.
```
Comment (kurtisvg) [inline sdks/langchain/README.md]: nit: suggest we wrap lines at ~80 characters. If you use VScode, rewrap is good for this: https://marketplace.visualstudio.com/items?itemName=stkb.rewrap

File: sdks/langchain/README.md [hunk 6]
```diff
@@ -0,0 +1,148 @@
...
+
+## Manual usage
+
+You can also execute a tool manually using the `arun` method.
+
+```python
+result = await tools[0].arun({ "name": "Alice", "age": 30 })
+```
+
+## Development
```
Comment (kurtisvg) [inline sdks/langchain/README.md]: I think this can move into a separate "DEVELOPER.md" similar to the top level project.

File: sdks/langchain/README.md [hunk 7]
```diff
@@ -0,0 +1,148 @@
+# GenAI Toolbox SDK
+
+This SDK allows you to seamlessly integrate the functionalities of [Toolbox](https://github.com/googleapis/genai-toolbox) into your LLM applications, enabling advanced orchestration and interaction with GenAI models.
+
+> [!NOTE]
```
Comment (kurtisvg) [inline sdks/langchain/README.md]: nit: A TOC would be nice. If you use VScode this extension will generate one for you: https://marketplace.visualstudio.com/items?itemName=huntertran.auto-markdown-toc

**Comment by kurtisvg** [comment]
@anubhav756 -- commit history is pretty messy, which makes this hard to follow. Could you rebase? (If you are unsure how, lmk and I'll walk you through it)

File: sdks/langchain/README.md (hunk 3 above)
Comment (anubhav756) [reply sdks/langchain/README.md:30]: Added a note. Thanks for pointing that out!

**Comment by anubhav756** [comment]
> @anubhav756 -- commit history is pretty messy, which makes this hard to follow. Could you rebase? (If you are unsure how, lmk and I'll walk you through it)

Done. Thanks for the suggestion!

File: sdks/langchain/README.md (hunk 6 above)
Comment (anubhav756) [reply sdks/langchain/README.md]: Thanks for the suggestion!

File: sdks/langchain/README.md [hunk 8]
```diff
@@ -0,0 +1,146 @@
...
+
+You can install the Toolbox SDK for LangChain using `pip`.
+
+```bash
+pip install toolbox-langchain-sdk
+```
+
+> [!IMPORTANT] This SDK is not yet available on PyPI. For now, install it from
+> source by following these [installation instructions](#development) (up to
+> step 3).
```
Comment (kurtisvg) [inline sdks/langchain/README.md:35]: nit: would move this to the top

File: sdks/langchain/README.md [hunk 9]
```diff
@@ -0,0 +1,146 @@
+# GenAI Toolbox SDK
+
+This SDK allows you to seamlessly integrate the functionalities of
+[Toolbox](https://github.com/googleapis/genai-toolbox) into your LLM
+applications, enabling advanced orchestration and interaction with GenAI models.
+
+> [!NOTE] Currently the SDK only supports
+> [LangChain](https://python.langchain.com/docs/introduction/).
```
Comment (kurtisvg) [inline sdks/langchain/README.md:8]: Doesn't look like this is rendering. Maybe try like this? Same for some of the others
```suggestion
> [!NOTE] 
> Currently the SDK only supports [LangChain](https://python.langchain.com/docs/introduction/).
```

File: sdks/langchain/README.md [hunk 10]
```diff
@@ -0,0 +1,146 @@
...
+## Installation
+
+You can install the Toolbox SDK for LangChain using `pip`.
+
+```bash
+pip install toolbox-langchain-sdk
+```
+
+> [!IMPORTANT] This SDK is not yet available on PyPI. For now, install it from
+> source by following these [installation instructions](#development) (up to
```
Comment (kurtisvg) [inline sdks/langchain/README.md:34]: This link looks wrong -- should it be:
```suggestion
> source by following these [installation instructions](#DEVELOPER.md) (up to
```

**Comment by anubhav756** [comment]
> LGTM but please fix comments

Thanks for pointing these out! Fixed in #54
